tests/unit/helpers/class-wc-mock-wc-data-store.php

```diff
@@ -109,7 +109,7 @@ public function update( &$object ) {
 		if ( 'user' === $this->meta_type ) {
 			wp_update_user(
 				[
-					'ID'         => $customer_id,
+					'ID'         => $customer_id, // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
```

Comment:
@shendy-a8c after adding the variable analysis sniff I'm seeing a warning here because `$customer_id` is undefined in the mock data store.

You added this 3 years ago (woah 😅 ) in #1148, I don't suppose you remember whether we can just add a phpcs ignore for this like I'e done or if it's something we should fix? I ask because I have no idea how to go about fixing this if we have to do that. Maybe the `$object` has a `customer_id` prop?